sympy/core/coreerrors.py

```diff
@@ -1,6 +1,5 @@
 """Definitions of common exceptions for :mod:`sympy.core` module. """
 
-from __future__ import print_function, division
 
 
```

Comment:
There are many cases like this where probably another blank line should be removed when the import is deleted

Reply:
Why is flake8 not responding to this? I guess this sort of test is not included.

Reply:
Hey, I was going through something else recently and stumbled across `black`. Guess what, it can do this automatically! Moreover, it can convert any messy python code to a strict PEP8 style code. I don't know whether this was discussed before or not.

Reply:
I'm not sure that we want to auto-style everything but it could be worth submitting a PR that shows what the effect of black is on a particular package so it can be discussed.